FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -740,7 +751,7 @@ protected function setProperties()
         // Since Shopware 5.3, attributes are assigned to the articles main details. Already existing ones aren't
         // touched.
         // \Shopware\Models\Article\Article::getAttribute has been removed in Shopware 5.6 entirely.
-        if (is_callable([$this->baseArticle, 'getAttribute']) && !is_null($this->baseArticle->getAttribute())) {
+        if (is_callable([$this->baseArticle, 'getAttribute']) && $this->baseArticle->getAttribute() !== null) {
```

Comment:
Since we are exporting the free-text fields now as attributes, we do no longer need them as properties.